### DISCUSSION TITLE
how do i get the current 1.18 build to open

### DISCUSSION THREAD

**Discussion by fletchermed1** [mircokroon/minecraft-world-downloader#334] [Q&A]
when i click the exe file for 1.18 it doesnt do anything is this happening to anyone else or only me i also tried the jar file but when i ran the command it said one of the class files was compiled in a newer version of java but i have the latest version of java.

**Comment by JorelAli** [comment]
> when i click the exe file for 1.18 it doesnt do anything is this happening to anyone else or only me i also tried the jar file but when i ran the command it said one of the class files was compiled in a newer version of java but i have the latest version of java.

What version of Java do you have installed on your machine? (You can find out by running `java -version` in a terminal/command prompt). This project requires Java 17 to run.
